programs/test/udp_proxy.c

```diff
@@ -613,11 +616,21 @@ int main( int argc, char *argv[] )
     int nb_fds;
     fd_set read_fds;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  ! mbedtls_platform_setup returned %d\n\n", ret );
+        goto exit;
+    }
+#endif
+
```

Comment:
Do we need platform initialization here? This is only a udp proxy, without any reason to have aplatform context here ( no cryptography, and only networking API)